ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -64,7 +77,7 @@ sub handle_config_file
         backup => '.old'
     );
 
-    foreach my $option_set (qw(comment_options options)) {
+    foreach my $option_set (qw(comment_options options multiline_options)) {
```

Comment:
what if we build a list of key/value first (adding the HostKey as key/value entries with same key and different), and then using that list to `add_or_replace_lines`?
```perl
my @todo;
foreach my $option_set (qw(comment_options options)) {
    next if (! $cfg->{$option_set});
    my $comment = $option_set eq 'comment_options' ? '#' : '';
    foreach my $option (sort keys %{$cfg->{$option_set}}) {
             my $val = $cfg->{$option_set}->{$option};
             if (grep {$_ eq $option} SSHD_CONFIG_MULTILINE) {
                 foreach my $ml_value (@$val) {
                     push(@todo, [$option, $ml_value, $comment]);
                 }
             } else {
                 push(@todo, [$option, $val, $comment]);
             } 
    }
}
foreach my $data (@todo) {
     my ($option, $val, $comment) = @$data;
     ... 
}
```